contracts/mocks/hts-precompile/HtsPrecompileMock.sol

```diff
@@ -0,0 +1,1839 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.9;
+
+import 'forge-std/console.sol';
+
+import '../../../contracts/hts-precompile/HederaResponseCodes.sol';
+import '../../../contracts/hts-precompile/KeyHelper.sol';
+import './HederaFungibleToken.sol';
+import './HederaNonFungibleToken.sol';
+import '../../../contracts/NoDelegateCall.sol';
+import '../../../contracts/libraries/Constants.sol';
+
+import '../interfaces/IHtsPrecompileMock.sol';
+import '../libraries/Validation.sol';
+
+contract HtsPrecompileMock is NoDelegateCall, KeyHelper, IHtsPrecompileMock {
```

Comment:
nit: We're trying to clarify precompile vs system contract.
If you get a chance
```suggestion
contract HtsSystemContractMock is NoDelegateCall, KeyHelper, IHtsPrecompileMock {
```